ajax/libs/pwnjs/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "pwnjs",
+  "filename": "pwn.min.js",
+  "description": "A Javascript library for browser exploitation",
+  "version": "1.0.0",
+  "keywords": [
+    "pwn",
+    "exploitation"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "https://github.com/theori-io/pwnjs.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
+        ]
+      }
+    ]
+  },
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/theori-io/pwnjs"
```

Comment:
I'll help let the url as the same with upstream https://github.com/theori-io/pwnjs/blob/master/package.json#L29